draft-bbs-signatures.md

```diff
@@ -318,19 +320,25 @@ Outputs:
 
 Procedure:
 
-1. (w, h0, h) = octets_to_point(PK)
+1. (W, H0, H) = octets_to_point(PK)
 
-2. e = H(PRF(8*ceil(log2(r)))) mod r
+2. h = SHAKE256(SK  || msg[i] || ... || msg[L])
```

Comment:
Assume this should actually be generalized here?
```suggestion
2. h = HASH(SK  || msg[i] || ... || msg[L])
```

Reply:
No its different.

Reply:
So you are introducing a dependency on a concrete hash algorithm here? I think previous conversations in the working group and documented in issues like #8 #46 #5 was to have this abstracted away?